docs/INSTALLATION.md

```diff
@@ -153,7 +153,8 @@ Daarnaast zijn er bronnen die binnen KISS doorzocht moeten worden.
 | `KLANTEN_BASE_URL`                      | URL van de Klanten API van het gebruikte klantenregister <details> <summary>Meer informatie </summary>Bijvoorbeeld `https://klantenregister.mijngemeente.nl/klanten` </details>  |

```

Comment:
In hoeverre vervallen alle `KLANTEN`- env vars nu, na overgang op OK2? Want: we gaan de klanten-api toch niet meer ondersteunen?

Reply:
Goed punt. Ik heb de verwijzingen naar de variabelen weggehaald. Overige aanpassingen in de handleiding van verwijzingen naar Klanten apis en contactmomenten apis (ipv partijen en klantcontacten) Kunnen we wat mij betreft beter in een los PR oppakken.